docs/languages/cpp.md

```diff
@@ -194,57 +143,77 @@ To go to a symbol's definition, place your cursor on the symbol anywhere its use
 
-* You can type an expression into the __Watch__ pane and it will be evaluated each time a breakpoint is hit.
+* You can type an expression into the __Watch__ section and it will be evaluated each time a breakpoint is hit.
 * You can type an expression into the __Debug Console__ and it will be evaluated only once.
 * You can evaluate any expression that appears in your code while you're stopped at a breakpoint.
 
-Note that expressions in the Watch Pane take effect in the application being debugged; an expression that modifies the value of a variable will modify that variable for the duration of the program.
+Note that expressions in the __Watch__ section take effect in the application being debugged; an expression that modifies the value of a variable will modify that variable for the duration of the program.
 
-### Multi-threaded debugging
+### Multi-threaded Debugging
 
-The C/C++ extension for VS Code now has the ability to debug threaded code.
+The C/C++ extension for VS Code has the ability to debug multi-threaded programs. All threads and their call stacks appear in the __Call Stack__ section:
 
-### Core Dump debugging
+![Multi-threaded process](images/cpp/threads.png)
 
-The C/C++ extension for VS Code also has the ability to debug using a memory dump. To debug using a memory dump, open your launch.json file for editing and add the `coreDumpPath` property to the __C++ Launch__ configuration, setting its value to be a string containing the path to the core dump. This will even work for multi-threaded programs and x86 programs being debugged on an x64 machine.
+### Memory Dump Debugging
 
-### GDB and MI commands
+The C/C++ extension for VS Code also has the ability to debug using a memory dump. To debug using a memory dump, open your `launch.json` file for editing and add the `coreDumpPath` (for GDB or LLDB) or `dumpPath` (for the Visual Studio Windows Debugger) property to the __C++ Launch__ configuration, setting its value to be a string containing the path to the memory dump. This will even work for multi-threaded programs and x86 programs being debugged on an x64 machine.
 
-You can execute GDB or MI commands directly through the debug console with the `-exec` command, but be careful -- executing GDB commands directly in the debug console is untested and might crash VS Code in some cases. For more information on debugging with VS Code, see this introduction to [debugging in VS Code](/docs/editor/debugging.md).
+### Additional Symbols
+
+If there are additional directories where the debugger can find symbol files (e.g., `.pdb` files for the Visual Studio Windows Debugger), they can be specified by adding the `additionalSOLibSearchPath` (for GDB or LLDB) or `symbolSearchPath` (for the Visual Studio Windows Debugger).
+
+For example:
+
+```json
+    "additionalSOLibSearchPath": "/path/to/symbols:/another/path/to/symbols"
+```
+or
+
+```json
+    "symbolSearchPath": "C:\\path\\to\\symbols;C:\\another\\path\\to\\symbols"

```

Comment:
can you add an example where the path has a space and you need to escape the quotes?